apps/devhub/templates/devhub/versions/add_file_modal.html

```diff
@@ -51,12 +51,12 @@
         <label>{{ new_file_form.beta }} {{ _('Only publish this version to my beta channel.') }}</label>
         <span class="tip tooltip" title="{{ new_file_form.beta.help_text }}">?</span>
         <p class="beta-warning"><em>{{ _('Warning:') }}</em>
-          {% trans addon_signing_url='https://wiki.mozilla.org/Addons/Extension_Signing', bugzilla_url='https://bugzilla.mozilla.org/enter_bug.cgi?product=addons.mozilla.org&component=Developer%20Pages' %}
+          {% trans addon_signing_url='https://wiki.mozilla.org/Addons/Extension_Signing', issues_url='https://github.com/mozilla/olympia/issues' %}
             The behavior of the beta channel has changed to accommodate
             <a href="{{ addon_signing_url }}" target="_blank">mandatory add-on signing</a>.
             The new process is currently in an open beta, and may change
             significantly in the near future. Please
-            <a href="{{ bugzilla_url }}" target="_blank">report any bugs</a> that you encounter.
+            <a href="{{ issues_url }}" target="_blank">report any bugs</a> that you encounter.
```

Comment:
I think this will cause this to be translated again. Perhaps it should stay as `bugzilla_url`?